cocos/spine/skeleton-cache.ts

```diff
@@ -338,7 +342,28 @@ class SkeletonCache {
         }
     }
 
-    public removeSkeleton (uuid: string): void {
+    public removeSkeleton (uuid: string, skeletonComponent: Skeleton): void {
+        const sharedInstances = this._sharedCacheMap.get(uuid);
+        if (sharedInstances) {
+            const index = sharedInstances.indexOf(skeletonComponent);
+            if (index !== -1) {
+                sharedInstances.splice(index, 1);
+            }
+            if (sharedInstances.length > 0) {
+                return;
+            }
+            this._sharedCacheMap.delete(uuid);
+        }
+
+        const sharedOperate = (aniKey: string, animationCache: AnimationCache): void => {
+            this._animationPool[`${uuid}#${aniKey}`] = animationCache;
+            animationCache.clear();
+        };
+        const privateOperate = (aniKey: string, animationCache: AnimationCache): void => {
```

Comment:
Why is it called `privateOperate`? What does `private` mean here?

Reply:
private corresponds to privateCache mode.